rspace/src/main/scala/coop/rchain/rspace/ReplayRSpace.scala

```diff
@@ -315,6 +315,16 @@ class ReplayRSpace[F[_]: Sync, C, P, A, K](
     comms.tailRecM(go).map(_.toOption)
   }
 
+  def spawn: F[IReplaySpace[F, C, P, A, K]] = {
```

Comment:
I think we should keep the idea of manager to spawn new instance like KeyValueStoreManger and RUntimeManager.